integrated function and change format

jenlcmc opened the pull request:
add remove_space() to sub + vinge cipher. Change format by using std:: instead of using namespace std. 

Still have to fix the error when compiling on mac. (ex: char user{} not working and most of caesar.cpp not working for mac) 
-> error (ld: warning: ignoring file caesar.o, building for macOS-x86_64 but attempting to link with file built for unknown-unsupported file format ( 0x7F 0x45 0x4C 0x46 0x02 0x01 0x01 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 )
Undefined symbols for architecture x86_64:
  "caesar::run_program()", referenced from:)

jenlcmc committed:
integrated function and change format

add remove_space() to sub + vinge cipher. Change format byusing std:: instead of using namespace std. Still have to fix error when compile on mac. (ex: char user{} not working and most of caesar.cpp not working for mac)

iamrelentless commented:
> add remove_space() to sub + vinge cipher. Change format by using std:: instead of using namespace std.
> 
> Still have to fix the error when compiling on mac. (ex: char user{} not working and most of caesar.cpp not working for mac)
> -> error (ld: warning: ignoring file caesar.o, building for macOS-x86_64 but attempting to link with file built for unknown-unsupported file format ( 0x7F 0x45 0x4C 0x46 0x02 0x01 0x01 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 0x00 )
> Undefined symbols for architecture x86_64:
> "caesar::run_program()", referenced from:)

Changed Variables that used {}, try testing on mac os env.

jenlcmc committed:
modify caesar to make it work with Clang compiler in macos

change few things in caesar file. Still need to work on whitespace and fix sub cipher error go to the end after enter key + mess

iamrelentless commented:
Changes made.

iamrelentless reviewed (commented):
Changes made.

iamrelentless reviewed (approved):
Merge the items. Changes have been made since opening.

jenlcmc committed:
Redo menu and add ASCII art

redo menu for main menu + ciphers and add ASCII art. add return to main menu from cipher's menu for user to choose other cipher instead of just stoping whole program

jenlcmc committed:
updates/organize files

-Add gitignore file
-Put h files into folder and change makefile
-Small change for cpp files

jenlcmc committed:
small change

add make clean and fix subcipher

jenlcmc committed:
Update README.md

jenlcmc committed:
change vigenere cipher

jenlcmc committed:
Update README.md

jenlcmc committed:
Merge branch 'main' into test

jenlcmc committed:
Merge branch 'test' of https://github.com/jenlcmc/cipher into test
